src/platform/forms-system/src/js/components/FormNav.jsx

```diff
@@ -1,14 +1,12 @@
-import React from 'react';
 import _ from 'lodash/fp'; // eslint-disable-line no-restricted-imports
+import React from 'react';
 import shallowEqual from 'recompose/shallowEqual';
-
-import SegmentedProgressBar from './SegmentedProgressBar';
-
 import {
   createFormPageList,
   createPageList,
   getActiveExpandedPages,
 } from '../helpers';
+import SegmentedProgressBar from './SegmentedProgressBar';
 
```

Comment:
I know you have a plugin that reorders the imports, but having it make these changes really just adds to the noise that reviewers have to sort though.

Reply:
Yeah sorry! My ESLint runs when I save.